packages/docs/docs/advanced/project-variables.mdx

```diff
@@ -34,5 +34,5 @@ Accessing the variables inside of a scene is through
 [`useScene()`](/api/core/utils#useScene):
 
 ```ts
-const circleFill = useScene().variables.get('circleFill');
+const circleFill = useScene().variables.get('circleFill', '');
```

Comment:
Should be some valid color like `'blue'` since an empty string will throw if you try to use it as a fill.